### PR TITLE
Fix #43 Snapchat icon

### LINKED ISSUE #43: Fix Snapchat icon svg
Currently the Snapchat icon svg appears like this on AdGuard DNS
Screenshot 2023-02-03 at 6 32 51 am

Here is what it should look like
snapchat-logo-svgrepo-com

### PR DESCRIPTION
**Pull request by jslawler**
New appearance: https://icons.adguard.org/icon?domain=snapchat.com

### CODE REVIEW

**Review by jellizaveta — CHANGES_REQUESTED** [review]
To rewrite the data about a company, it is necessary to enter the information in the `adguard_companies.json` file.

File: dist/companies.json [hunk 1]
```diff
@@ -9303,7 +9303,7 @@
         },
         "snap_technologies": {
             "name": "Snap Technologies",
-            "websiteUrl": "http://www.snap.com/",
+            "websiteUrl": "http://www.snapchat.com/",
```
Comment (jellizaveta) [inline dist/companies.json:9306]: `companies.json` contains companies list merged with AdGuard companies from `adguard_companies.json`,  so the changes can only be added to `adguard_companies.json`, if I understand correctly

**Review by jellizaveta — CHANGES_REQUESTED** [review]

File: dist/companies.json (hunk 1 above)
Comment (jslawler) [reply dist/companies.json:9306]: I'll do it and we'll see what happens. :)

File: dist/companies.json [hunk 2]
```diff
@@ -9301,11 +9301,6 @@
             "websiteUrl": "https://snapengage.com",
             "description": "SnapEngage is a live, in-browser chat solution. Their product allows websites to quickly and proactively engage prospective customers."
         },
-        "snap_technologies": {
```
Comment (ameshkov) [inline dist/companies.json:9304]: Don't change this file at all, it will be overriten automatically.

**Review by ameshkov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**